djstripe/event_handlers.py

```diff
@@ -114,23 +114,51 @@ def customer_subscription_webhook_handler(event):
     _handle_crud_like_event(target_cls=models.Subscription, event=event)
 
 
+@webhooks.handler("payment_method")
```

Comment:
Is it still possible to hook this into "payment_method.detached" directly?

Reply:
My understanding was that if we did that then we'd need to register `"payment_method"` to ``other_object_webhook_handler``, which would mean ``payment_method.detached`` would go to both, and hit an error in ``other_object_webhook_handler``.